<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>goodClass</class>
 <widget class="QMainWindow" name="goodClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>good</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더를 입력하세요.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="imageList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>171</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>실행 결과</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_result">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>360</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>이전</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>총 파일 수 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileSize">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_car">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>자동차</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_man">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>사람</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_face">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>얼굴</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>80</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="good.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>goodClass</receiver>
   <slot>openButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevButton</sender>
   <signal>clicked()</signal>
   <receiver>goodClass</receiver>
   <slot>prevImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>goodClass</receiver>
   <slot>nextImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>goodClass</receiver>
   <slot>selectedImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>processButton</sender>
   <signal>clicked()</signal>
   <receiver>goodClass</receiver>
   <slot>processButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openButton()</slot>
  <slot>pushButton()</slot>
  <slot>prevImage()</slot>
  <slot>nextImage()</slot>
  <slot>selectedImage()</slot>
  <slot>processButton()</slot>
 </slots>
</ui>
